### DISCUSSION TITLE
how to fix this error nextjs14 for use --turbo: Error: EPERM: operation not permitted in

### DISCUSSION THREAD

**Discussion by peymanath** [vercel/next.js#57573] [App Router]
Hello everyone,
i'm using Turbo in the developer environment nextjs14, but I'm getting the following error.
Has anyone encountered this error?

Note: When I get this error that NEXTjs wants to render my routes.

```bash


   ▲ Next.js 14.0.0 (turbo)
   - Local:        http://localhost:3000

 ✓ Ready in 1161ms
 ○ Compiling /branchs/page ...
 ✓ Compiled /branchs/page in 3.9s
 ○ Compiling /_error ...
 ✓ Compiled /_error in 766ms
 ⨯ [Error: EPERM: operation not permitted, rename 'D:\Work\Development\Project\FrontEnd\Terxine\terxine\.next\build-manifest.json.tmp.jazqmzy5ees' -> 'D:\Work\Development\Project\FrontEnd\Terxine\terxine\.next\build-manifest.json'] {
  errno: -4048,
  code: 'EPERM',
  syscall: 'rename',
  path: 'D:\\Work\\Development\\Project\\FrontEnd\\Terxine\\terxine\\.next\\build-manifest.json.tmp.jazqmzy5ees',
  dest: 'D:\\Work\\Development\\Project\\FrontEnd\\Terxine\\terxine\\.next\\build-manifest.json'
}


```

**Comment by tbenci** [comment]
It is doing it out of the box with a simple create-next-app@latest. Makes the whole point of updating to 14 (turbo) worthless.

**Comment by ChetSocio** [comment]
Take a look at: [this issue ](https://github.com/vercel/next.js/issues/57581).  They seem to have solved it already but but provided description of how they solved it  (and majorly how can we solve it too) and why it occured (in first place)  plus we are still getting this error after nextjs 14.0.1.  The problem is with turbopack or either nextjs app. I have never faced this problem in bare reactjs app in any way. Its coming only when we use turbo with nextjs.

**Comment by LucaBlackDragon** [comment]
I had a very similar issue.
In my case, the issue was caused by my antivirus (Bitdefender Total Security 27.x).
I solved it by adding an exception for the project folder and deactivating the Embedded scripts protection:
![image](https://github.com/vercel/next.js/assets/3714010/f553e104-80d7-4956-9401-82eeb6f9b288)

**Comment by Hayelalfaisali** [comment]
For me, it works fine this answer drives me to the resolver. I run 2 scripts in terminals:

once for development
another for the build process.
So after killing the development script it looks perfectly fine!

**Comment by Hayelalfaisali** [comment]
Check host file which like this

##
# Host Database
#
# localhost is used to configure the loopback interface
# when the system is booting.  Do not change this entry.
##
127.0.0.1   localhost
255.255.255.255 broadcasthost

**Reply by Mohammed-abdullah-abdullah** [reply]
thanks it works for me perfectly
